websites/docs-smoke-test/src/release-notes/bugfix.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Release notes not reappearing when changed
+date: 2020-03-16
+type: Resolved Issue
+topics:
+  - Settings
+description: |
+  No more old release notes popping up in external feeds, we found the bug.
+---
+
+ * Authors can change old releases now, for example to fix spelling errors without the release note popping up again in feed consumers
+ * Release Notes have an autogenerated unique ID now.
```

Comment:
Some questions regarding this:
- what are the requirements for this ID?
- where/what is the ID supposed to be used for?

Reply:
WIP, this is basically lorem ipsum.   Can't revert a PR back to draft status,  @davifantasia  created it as an open PR directly.